packages/cli/src/index.ts

```diff
@@ -464,6 +470,11 @@ program
         'Force upload even if local files have not changed, use this when you want to upload files to a new project',
         false,
     )
+    .option(
+        '--path <path>',
```

Comment:
should we add the `-p` option too ? 

![image](https://github.com/user-attachments/assets/4337ec88-4ed7-4681-a1d7-b29c42a02a3b)